src/SimulationService.jl

```diff
@@ -218,14 +218,14 @@ end
 
 #-----------------------------------------------------------------------------# OperationRequest
 Base.@kwdef mutable struct OperationRequest
-    obj::JSON3.Object = JSON3.Object()                      # untouched JSON from request sent by HMI
-    id::String = "sciml-$(UUIDs.uuid4())"                   # matches DataServiceModel :id
-    route::String = "unknown"                               # :simulate, :calibrate, etc.
-    model::Union{Nothing, JSON3.Object} = nothing           # ASKEM Model Representation (AMR)
-    models::Union{Nothing, Vector{JSON3.Object}} = nothing  # Multiple models (in AMR)
-    timespan::Union{Nothing, NTuple{2, Float64}} = nothing  # (start, end)
-    df::Union{Nothing, DataFrame} = nothing                 # dataset (calibrate only)
-    result::Any = nothing                                   # store result of job
```

Comment:
What is the diff here?  I don't see any changes, right?

Reply:
I think I changed something here, then changed it back, but didn't change the spacing back. I'll fix that.